lib/bemxjst/index.js

```diff
@@ -426,13 +426,13 @@ BEMXJST.prototype.tryRun = function(context, ent) {
   try {
     return ent.run(context);
   } catch (e) {
-    console.error('BEMXJST ERROR: cannot render ' +
-      [
-        'block ' + context.block,
-        'elem ' + context.elem,
-        'mods ' + JSON.stringify(context.mods),
-        'elemMods ' + JSON.stringify(context.elemMods)
-      ].join(', '), e);
+    var error = this.context.onError || console.error;
```

Comment:
Guess we don't really need this variable: `return (this.context.onError || defaultErrorHandler)(context, e) || '';`